indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+                  Do you want to be amongst the first space flight passengers
+                  to Mars? Join our flight service (as endorsed by Patrick
+                  Mooreish) and see the Red Planet for yourselves !!
+                  All you have to do is fill in the details (Name, Address,
+                  Phone, Email and Credit Card details) and we will rush you
+                  the tickets. This flight is SO NOT likely to happen !!!!!! (for a while)</p>
+                  </div>  <!-- end col2content -->
+            </div>  <!-- end col2 -->
+            <div id="footer">
+            <p><b>&copy; Bear Computer Services Ltd. | Original style design by BlueSky Web Studios</b></p>
+            </div>  <!-- end footer -->
+      </div>  <!-- end wrapper -->
+</body>
+<script>
+		 function Init()
+		 {
+		  	var ins = document.getElementsByTagName("input");
+		    var imgs = document.getElementsByTagName("img");
+		 
+	 		if (window.addEventListener)
+	 		{
+	 			for(i=0 ; i<ins.length ; i++){
+						ins[i].addEventListener("focus",doHighlights,false);
+	 					ins[i].addEventListener("blur",doHighlights,false);
+				}
+				
+	 			document.getElementById("blurb").addEventListener("mouseover",doBackground,false);
+	 			document.getElementById("blurb").addEventListener("mouseout",doBackground,false);
+				
+				for(i=0 ; i<imgs.length ; i++){
+	 					imgs[i].addEventListener("mouseover",doText,false);
+	 					imgs[i].addEventListener("mouseout",doText,false);
+				}
+	 		}
+	 		else if (window.attachEvent)
+	 		{
+	 			for(i=0 ; i<ins.length ; i++){
+						ins[i].attachEvent("focus",doHighlights);
+	 					ins[i].attachEvent("blur",doHighlights);
+				}
+	 			document.getElementById("blurb").attachEvent("onmouseover",doBackground);
+	 			document.getElementById("blurb").attachEvent("onmouseout",doBackground);
+				for(i=0 ; i<imgs.length ; i++){
+	 					imgs[i].attachEvent("mouseover",doText);
+	 					imgs[i].attachEvent("mouseout",doText);
+				}
+	 		}
+	 		else
+	 		{
+ 			 	alert("I have nothing");
+	 		}
+	 	}
+		function doHighlights(e)
+		{
+			if (!e) { e = window.event; }
+			if (e.type == "focus" || e.type == "onfocus"){
+		 		 document.getElementById(this.id).style.backgroundColor = "red";
+			} else {
+				 document.getElementById(this.id).style.backgroundColor = "white";
+			}
+		}
+		function doBackground(e)
+		{
+			if (!e) { e = window.event; }
+			if (e.type == "mouseover" || e.type == "onmouseover"){
+		 		 document.getElementById("blurb").style.backgroundColor = "yellow";
+			} else {
+				 document.getElementById("blurb").style.backgroundColor = "";
+			}
+		}
+		function doText(e)
+		{
+			if (!e) { e = window.event; }
+			if (e.type == "mouseover" || e.type == "onmouseover"){
+		 		 document.getElementById(this.id + "T").innerHTML = document.getElementById(this.id).alt;
+			} else {
+				 document.getElementById(this.id + "T").innerHTML = "&nbsp;";
+			}
+		}
+		
+	 Init()
+
+</script>

```

Comment:
It's really good to see all of the code together, but it may be easier to read and maintain if the JavaScript and HTML were stored in separate files.